### ISSUE TITLE
Missing Hisui Gen 8 data

### ISSUE THREAD

**Issue by ZebTheWizard** [PokeAPI/pokedex#89]
The API is missing new Pokémon from the Hisui region from the new game Pokémon Legends: Arceus.  Data for `NDex` 899-905 are missing.

**Comment by Naramsim** [comment]
We don't really have data for Pokémon Legends: Arceus.

It came out a few days ago, right?

**Comment by ZackBz** [comment]
Yeah, it came out a week ago today. Any idea when the data will be added to the API?

**Comment by Naramsim** [comment]
Hi @ZackBz, most probably when some contributors will start adding it. I don't see Veekun adding it soon, so we will have to scrape it and add it by ourselves. In the past for Sword and Shield we had great contributions from many people, we will see if it will be the same for the new update

**Comment by anhthang** [comment]
Adding in #95
